code/__HELPERS/unsorted.dm

```diff
@@ -1366,6 +1366,13 @@ proc/find_holder_of_type(var/atom/reference,var/typepath) //Returns the first ob
 /proc/print_runtime(exception/e)
 	world.log << "[time_stamp()] Runtime detected\n[e] at [e.file]:[e.line]\n [e.desc]"
 
+/proc/transfer_fingerprints(atom/A,atom/B)
+	if(!A || !B)
+		return
+	B.fingerprints = A.fingerprints
```

Comment:
Thanks for the explanation. Though in the case of the paintings, it works out since they are meant to be the same item (both the structure on the wall and the frame that you pick up).
If there is already another proc that copies fingerprints, then we can keep this one to "link" the fingerprints of two atoms.